code/style.css

```diff
@@ -1,5 +1,84 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
 body {
-  background-color: lightblue;
+  font-family: 'Quicksand', sans-serif;
+  font-style: normal;
 }
+
+.grid-container{
+  height: 100vh;
+  width: 100vw;
+  display: grid;
+  grid-template-columns: repeat(2 1fr);
+  grid-template-rows: 50px auto auto auto 50px;
+}
+.menu-box {
+  background-color: #D7DBDD;
+  grid-column: span 2;
+  grid-row: 1;
+}
+menu {
+  display: flex;
+  justify-content: center;
+}
+menu a {
+  display: inline-block;
+  padding: 0 15px;
+  background: #D7DBDD;
+  color:black;
```

Comment:
Minor things, but:
- be consistent and add some space between the element and the opening curly bracket: `.grid-container {`
- same between CSS property and value: `color: black;`
- add line breaks between the elements you're styling:

```
.grid-container {
  height: 100vh;
  width: 100vw;
  display: grid;
  grid-template-columns: repeat(2 1fr);
  grid-template-rows: 50px auto auto auto 50px;
}

.menu-box {
  background-color: #D7DBDD;
  grid-column: span 2;
  grid-row: 1;
}

menu {
  display: flex;
  justify-content: center;
}

menu a {
  display: inline-block;
  padding: 0 15px;
  background: #D7DBDD;
  color: black;
```